unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2928,6 +2945,45 @@ abstract public class BaseFPSAgentController {
             }
         }
 
+        public void GetVisibleObjects(float? maxDistance = null, string visibilityScheme = null) {
+            VisibilityScheme visSchemeEnum;
+            if (visibilityScheme != null) {
+                visibilityScheme = visibilityScheme.ToLower();
+                if (visibilityScheme.ToLower() == "collider") {
```

Comment:
Change string literal comparison to -> `Enum.GetName(typeof(VisibilityScheme), VisibilityScheme.Collider, ignoreCase: true) == visibilityScheme `

Reply:
Results in an error:
```
Assets/Scripts/BaseFPSAgentController.cs(2956,107): error CS1739: The best overload for 'GetName' does not have a parameter named 'ignoreCase'
```

Will do
```
Enum.GetName(typeof(VisibilityScheme), VisibilityScheme.Collider).ToLower() == visibilityScheme
```
instead.